packages/styles/notice.css

```diff
@@ -0,0 +1,66 @@
+:root {
+  --notice-info-color: var(--accent-info-active);
+  --notice-caution-color: var(--accent-caution);
+  --notice-text-color: var(--accent-dark);
+  --notice-title-text-color: var(--accent-dark);
+  --notice-title-font-weight: var(--font-weight-medium);
+  --notice-background-color: var(--notice-info-color);
+  --notice-border-color: var(--accent-dark);
+}
+
+.cauldron--theme-dark {
+  --notice-text-color: var(--accent-dark);
+  --notice-title-text-color: var(--accent-dark);
+}
```

Comment:
Hmm, are you sure? Running into issues with links and headings in particular. Or are you just saying can use the same variables for both since they're not changing?